src/parser/jobs/rdm/modules/Procs.tsx

```diff
@@ -38,6 +42,20 @@ export class Procs extends CoreProcs {
 			procStatus: this.data.statuses.VERFIRE_READY,
 			consumeActions: [this.data.actions.VERFIRE],
 		},
+		{
+			procStatus: this.data.statuses.GRAND_IMPACT_READY,
+			consumeActions: [this.data.actions.GRAND_IMPACT],
+		},
+		//We are uncertain if we want this here or on their own, for now they'll be on their own.
+		//Leaving this here in case we change our minds
+		// {
+		// 	procStatus: this.data.statuses.PREFULGENCE_READY,
+		// 	consumeActions: [this.data.actions.PREFULGENCE],
+		// },
+		// {
+		// 	procStatus: this.data.statuses.THORNED_FLOURISH,
+		// 	consumeActions: [this.data.actions.VICE_OF_THORNS],
+		// },
```

Comment:
Is the only reason to separate these out to have separate timeline rows for them?  If so, would we be better off to add a property to CoreProcs to allow consumers to choose to show the Procs rows expanded by default (set up line 260 in CoreProcs to use collapse: this.collapseProcsRowsInTimeline and then let consumers override that to false if they have multiple distinct procs they want to show separated by default)?

I think the UI would be much better with putting in a way to show the Procs group as uncollapsed by default, rather than having 3 rows that all just are titled "Procs" from the different modules...

Roughly after (aside from all of the "Procs" headers being one header for all 4 rows):
![image](https://github.com/user-attachments/assets/d4fd97dc-64b8-4754-ae1c-abe796f7e5de)

As is with separated modules:
![image](https://github.com/user-attachments/assets/744f8ee6-b3b8-42fe-95ef-7f8d6136b568)

If we do keep separated modules, remove this commented code (we can always re-add it later).

Reply:
Correct, Thorned Flourish and Prefulgence are 2 of the highest potency skills, and the feedback I got was that those might be better served as having their own timeline rows.  So I commented out the code that had them in the base Procs module in case they changed their minds after having it for a while.  The intention is, whatever they decide I'll cleanup the method that wasn't wanted.  Since I already had code for both ways, I decided to just comment it out so it'd be a quick change if they decide to change it to all in one, or moving Grand Impact out into it's own row and keeping VerStone/VerFire under 1 proc.

As for the UI change for core; my thought would be as part of the proc declaration have title, and ownRow or some other bool named to denote it'll get it's own row.  Then on the if (this.showProcTimelineRow check for that ownRow bool and if so, add a new simple row with no collapse using the provided label; otherwise do what it does today.  The idea being it'd appear separate, not have the collapse bit and just look like any other tracked timeline skill.